cmd/dockerd/daemon.go

```diff
@@ -616,6 +616,10 @@ func loadDaemonCliConfig(opts *daemonOptions) (*config.Config, error) {
 		conf.CDISpecDirs = nil
 	}
 
+	if err := loadCLIPlatformConfig(conf); err != nil {
```

Comment:
Thanks! Yup, I think this is the cleanest way to do this indeed, as it keeps it internal.